website/assets/js/components/js-timestamp.component.js

```diff
@@ -20,7 +20,8 @@ parasails.registerComponent('jsTimestamp', {
   props: [
```

Comment:
I'm ok with these changes for fleetdm.com, but let's not bring them upstream to sails-generate

(unless you find yourself with some spare time and want to try reenvisioning the props and PR separately some time.  They're too crowded as it is-- and I think the `short` prop needs to go away)